packages/charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -143,7 +143,8 @@ export class LineChartBase extends React.Component<
     if (this.props.parentRef) {
       setTimeout(() => {
```

Comment:
This timeout can be executed while a component is unmounted, which will cause setState to execute in a bad situation. A better approach is to track the id and cancel on unmount. (you can cancel something which is already canceled, so just doing it always would be fine.)

Also perhaps `requestAnimationFrame` would be a better option.